clinica/pipelines/statistics_volume/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "id": "aramislab/statistics_volume",
+    "author": "John Doe",
+    "version": "0.1.0",
+    "dependencies": [
+        {
+            "type": "binary",
+            "name": "clinica",
+            "version": ">=0.1.0"
+        }
+    ]
+}
```

Comment:
Your `info.json` should look like:
```JSON
{
    "id": "aramislab/statistics-volume",
    "author": " Arnaud Marcoux",
    "version": "0.1.0",
    "space_caps": "42M",
    "space_wd": "42M",
    "dependencies": [
        {
            "type": "software",
            "name": "spm",
            "version": ">=12"
        }
    ]
} 
```
Don't forget to add adequate values for space_caps/space_caps.

Reply:
I don't know if matlab is checked together with SPM but this must be also checked.

Reply:
spm standalone does not use matlab

Reply:
Indeed. I added the task "Make check_spm also checks Matlab when using SPM as a Matlab toolbox." to make things clear.